### PR TITLE
Calculator for calculating the annual net and gross salary

### LINKED ISSUE #1: Task description
PaymentCalculator

### PR DESCRIPTION
**Pull request by wojciechkostecki**
Closes #1

### CODE REVIEW

File: src/PaymentCalculator.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+public class PaymentCalculator {
+
+    public double nettoYearPayment(Employee employee) {
+        return employee.salary * 12;
+    }
+
+    public double bruttoYearPayment(Employee employee) {
+        double zus = 1000;
```
Comment (rbobko93) [inline src/PaymentCalculator.java]: This should be final field of this class.

Comment (rbobko93) [reply src/PaymentCalculator.java]: Same with 0.2 which is tax. They don't change ever, so let's make then a final fields.

**Review by rbobko93 — CHANGES_REQUESTED** [review]

File: src/PaymentCalculator.java (hunk 1 above)
Comment (wojciechkostecki) [reply src/PaymentCalculator.java]: Done.

File: src/PaymentCalculator.java [hunk 2]
```diff
@@ -1,11 +1,12 @@
 public class PaymentCalculator {
+    final double zus = 1000;
```
Comment (rbobko93) [inline src/PaymentCalculator.java]: What is this not private? Remember about ACCESS MODIFIERS!

Comment (rbobko93) [reply src/PaymentCalculator.java]: Also fix the naming for final fields

Comment (rbobko93) [reply src/PaymentCalculator.java]: https://www.javatpoint.com/java-naming-conventions

**Review by rbobko93 — CHANGES_REQUESTED** [review]

File: src/PaymentCalculator.java [hunk 3]
```diff
@@ -1,12 +1,12 @@
 public class PaymentCalculator {
-    final double zus = 1000;
-    final double tax = 0.2;
+    private final static double ZUS = 1000;
```
Comment (rbobko93) [inline src/PaymentCalculator.java:2]: Nobody said to make them static, but that fine because they are not object dependent.

**Review by rbobko93 — APPROVED** [review]

**Pull request merged**